### ISSUE TITLE
item #4 text under "How It Works" on the front page is misleading

### ISSUE THREAD

**Issue by csggene3** [vacekj/regenscore#112]
![Screenshot 2023-10-06 000016](https://github.com/vacekj/regenscore/assets/10295641/dc9ada26-abc9-41ea-b74c-a9637a7f2335)

The link text says to share feedback and suggestions but the link is to a type form only for new attestation item suggestions and not regarding general feedback, bug reporting, etc.
Should either change the wording to reflect what the actual type form is and remove feedback, or add another form for feedback with a separate link.

**Comment by vacekj** [comment]
Good catch, we'll split it up into a new behaviour item suggestion link & a bug report/general feedback (GH Issues).

**Comment by vacekj** [comment]
share feedback - gh issues, behaviour item suggestions link - to form.
